bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -53,6 +55,16 @@ class CommandLineArguments
                         this->port = std::stoi(std::wstring(argv[index]));
                     }
                 }
+                else if (!arg.compare(L"--script"))
+                {
+                    ++index;
```

Comment:
Need to verify that there's another parameter so we don't read out of bounds.

Reply:
Will fix. For both --port and --script, I reversed the expression to use `if (index < argc)` for better readability.